File: Pythom/1. Elementos Basicos/Ejercicio #1.py
#1. Escribir la siguiente exprecion en forma de expresión algebraica

print("Ecuacion: a³*(b²-2ac) / 2b")
a = float(input("Digite el valor de a: "))
b = float(input("Digite el valor de b: "))
c = float(input("Digite el valor de c: "))

RES = (a**3 * (b**2 - 2*a*c)) / (2*b)

print("El resultado es: ",RES)

File: 1. C++/2. Expresiones/Ejercicio #1.cpp
/*
	1. Escribe la siguiente exprecion como exprecion en C++
*/

#include<iostream>

using namespace std;

int main()
{
	float a, b, res = 0;
	
	cout<<"Digite el valor de la variable 'a': "; cin>>a;
	cout<<"Digite el valor de la variable 'b': "; cin>>b;
	
	res = (a/b)+1;
	
	cout<<"El resultado es: "<<res; 
	return 0;
}